### DISCUSSION TITLE
js seems ro interprete string as a date

### DISCUSSION THREAD

**Discussion by samy38** [vsdudakov/fastadmin#46] [Q&A] [answered]
Hello,

Your application is very imprecive!!!

I tried it but i have a problem:
I'm using fastapi+sqlalchemy

I have 2 columns in my model:
    identifier: Mapped[str] = mapped_column(String(length=255), unique=True)
    name: Mapped[str] = mapped_column(String(length=255), default="")

They are defined as string.
But in the dashboard/model the values are showed as datetime: 2001_0_1 00:00
In the detail i have another format: Sun, 31 Dec 2000 23:00:00 GMT
the api call are returning a json with the correct value: "1"

Can u telle me what i m doing wrong or how can i fix this?

**Comment by vsdudakov** [comment]
I suppose this is a frontend bug. Let me reproduce it and fix.

**Reply by vsdudakov** [reply, marked as answer]
@samy38 fixed here https://github.com/vsdudakov/fastadmin/releases/tag/0.2.7

**Reply by samy38** [reply]
thanks!! :)
